submissions/benjaminKamena/JavaScript/index.js

```diff
@@ -28,3 +28,41 @@ for(let i = 0; i< productList.length; i++){
 }
 
 //TASK 2
+
+// Writing the numbers from 1 and the positive difference between 7 and 123 
+var output = Math.abs(7-123);
+function diffNum(){
```

Comment:
You need to call the function for the output @kamena1994

Reply:
You need to call the function for your code to be executed